sale_double_validation/models/sale.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+from odoo import api, fields, models
+
+
+class SaleOrder(models.Model):
+    _inherit = 'sale.order'
+
+    state = fields.Selection(
+        selection_add=[('to_approve', 'To Approve')]
+    )
+
+    @api.multi
+    def is_amount_to_approve(self):
+        currency = self.env.user.company_id.currency_id
+        limit_amount = self.company_id.so_double_validation_amount
+        limit_amount = currency.compute(limit_amount, self.currency_id)
+        return self.amount_total >= limit_amount
```

Comment:
Comparison of floats should preferably be done with `odoo.tools.float_compare()`.